bin/README

```diff
@@ -0,0 +1,94 @@
+----------------
```

Comment:
why do you copy this from the root directory ?

Reply:
Mistake I guess. i will delete it from the bin. Might have to do something with Eclipse.

Reply:
Still not deleted ?